src/toxic.c

```diff
@@ -1573,6 +1577,19 @@ static void init_default_data_files(void)
     free(user_config_dir);
 }
 
+static Toxic *toxic_init(Tox *tox)
```

Comment:
I guess a lot of these things are leaked by design, but it would be nice to be able to return 0 from main and get a list of actual leaks by having it break out of the do_toxic loop and `kill` the things `init`'d in the far future. :P

Reply:
Yeah, we can put that on a todo list.